seed.d/rules.json

```diff
@@ -3,35 +3,35 @@
         "description":"This instance is sized properly but it is suffering from pressure.",
-        "reason": "Based on the host metrics set, we have detected that this {system.instance_type} instance on {system.cloud_provider.upper()} region {system.region} is correctly sized according to its utilization but it is experiencing bottlenecks, probably due to peaks.{newline}The specific issues identified by ROS are:{newline}{summary}",
-        "resolution": "Please pay attention to the specific pressure conditions (CPU, memory and/or disk I/O) and try to identify the hours where peaks are happening and why. You may use logs, your PCP tools or third party application specific tools (an APM, for instance). You may also want to consider changing the instance type from {system.instance_type} ({instance_price}) to a larger one in the dimensions where pressure was identified: {candidate_string}.",
+        "reason": "Based on the host metrics set, we have detected that this {system.instance_type} instance on {system.cloud_provider.upper()} region {system.region} is correctly sized according to its utilization but it is experiencing bottlenecks, probably due to peaks.",
+        "resolution": "Please pay attention to the specific pressure conditions (CPU, memory and/or disk I/O) and try to identify the hours where peaks are happening and why. You may use logs, your PCP tools or third party application specific tools (an APM, for instance).",
         "condition":"System is optimized but Kernel PSI is detected"
     },
     {
         "rule_id": "ros_instance_evaluation|STORAGE_RIGHTSIZING",
         "description":"This instance is not sized properly so consider enhancing the storage capabilities.",
-        "reason": "Based on the host metrics set, we have detected that this {system.instance_type} instance on {system.cloud_provider.upper()} region {system.region} is running into the storage I/O bottleneck.{newline}The specific issues identified by ROS are:{newline}{summary}",
-        "resolution": "Please pay attention to attached Storage of the RHEL instance, either utilizing your PCP tools or third party application specific tools to verify that the sizing of storage for this ordered instance type from provider is optimized for your workload.{newline}You may also want to consider changing the instance type from {system.instance_type} ({instance_price}) to one of the following instance types, which fit the load better: {candidate_string}.",
+        "reason": "Based on the host metrics set, we have detected that this {system.instance_type} instance on {system.cloud_provider.upper()} region {system.region} is running into the storage I/O bottleneck.",
+        "resolution": "Please pay attention to attached Storage of the RHEL instance, either utilizing your PCP tools or third party application specific tools to verify that the sizing of storage for this ordered instance type from provider is optimized for your workload.",
```

Comment:
Have discussed with @r14chandra that updating this file will not need DB migration as this is a static file 👍🏻

Reply:
Ohh okay got it. Thank you for sharing the detailed information Kavita!!

Reply:
I would like to correct myself here.
From @r14chandra's observation, data updated as expected in rules table without any manual deletion on stage.
@PreetiW  - No additional action needed for PROD too.